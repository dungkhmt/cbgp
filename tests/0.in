10 10
10 23
10 7
2 10
6 1
1 9
1 5
6 10
10 4
1 3
4 3
1 10
6 7
5 6
5 7
10 9
9 2
3 2
3 10
4 2
3 8
2 1
4 8
7 1
7 3
