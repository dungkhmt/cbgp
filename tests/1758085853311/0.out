(9, 6), (1, 9), (5, 0), (0, 8), (0, 4), (5, 9), (9, 3), (0, 2), (3, 2), (0, 9), (5, 6), (4, 5), (4, 6), (9, 8), (8, 1), (2, 1), (2, 9), (3, 1), (2, 7), (1, 0), (3, 7), (6, 0), (6, 2), 
numberIntersection=0 
minAngle=0.05123716740341866 
sumAngle=5.859322136205028 
minEdgeLength=1.0 
sumEdgeLength=135.37551107414222 
minDistance=0.25607375986579195 
sumDistance=928.8155877485585 

0.0000000000 0.0512371674 5.8593221362 1.0000000000 135.3755110741 0.2560737599 928.8155877486
0: (10, 3),
1: (13, 2),
2: (1, 1),
3: (16, 0),
4: (7, 5),
5: (7, 6),
6: (5, 4),
7: (0, 0),
8: (11, 4),
9: (8, 8),
10 3 13 2 1 1 16 0 7 5 7 6 5 4 0 0 11 4 8 8 
0: (10, 3), 1: (13, 2), 2: (1, 1), 3: (16, 0), 4: (7, 5), 5: (7, 6), 6: (5, 4), 7: (0, 0), 8: (11, 4), 9: (8, 8), 
numberIntersection=0 minAngle=0.05123716740341866 sumAngle=5.859322136205028 minEdgeLength=1.0 sumEdgeLength=135.37551107414222 minDistance=0.25607375986579195 sumDistance=928.8155877485585 
Time taken: 421.529 s
